src/NixFromNpm/Conversion/ToDisk.hs

```diff
@@ -179,11 +180,9 @@ initializeOutput = do
       -- Get the path to the files bundled with nixfromnpm which contain
       -- nix libraries.
       nodeLibPath <- (</> "nodeLib") <$> getDataFileName "nix-libs"
-      -- Copy each of these into the target (unless they exist).
-      forItemsInDir_ nodeLibPath $ \path -> do
-        whenM (isFile path) $ do
-          unlessExists (outputPath </> "nodeLib" </> filename path) $
-            copyFile path (outputPath </> "nodeLib" </> filename path)
+
+      Turtle.cptree nodeLibPath (outputPath </> "nodeLib")
```

Comment:
We already have `cp_r` imported from `shelly`; is there a compelling reason to add a new dependency for this behavior?

Reply:
I'm not super worried about the clobbering, but we might want to `rm` the previous `nodeLib` directory so that unused files are not left behind. Of course this could be a problem if people put their own custom files in there. Not sure what the best way to go about that is

Reply:
I wasn't aware of `cp_r`. I'll go ahead and use that to eliminate the extra dep.

I think removing `nodeLib` is fine.